src/bin/mkksiso

```diff
@@ -511,7 +511,11 @@ def MakeKickstartISO(input_iso, output_iso, ks="", add_paths=None,
         for p in add_paths:
             cmd.extend(["-map", p, os.path.basename(p)])
 
-        if CheckBigFiles(add_paths):
+        # include updates image which will be automatically loaded when on correct place
+        if updates_image:
+            cmd.extend(["-map", updates_image, "updates/updates.img"])
+
+        if CheckBigFiles(add_paths + [updates_image]):
```

Comment:
You need to check if updates_img is empty, otherwise you end up with `[""]` (which the tests caught, woo hoo!)

Reply:
Should be fixed. Feel free to say if you don't like to have this kind of expression. I like it but might be harder to read for someone.

Reply:
I usually like those, but I think having it as an argument to a function call makes the function it harder to read. Even just moving it up and giving it a new variable:

```python
check_paths = add_paths if not updates_image else add_paths + [updates_image]
if CheckBigFiles(check_paths):
```
is easier, I think.